USFMConverter/Core/CoreConverter.cs

```diff
@@ -2,29 +2,36 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Net;
+using System.Runtime.InteropServices.ComTypes;
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
 using USFMConverter.Core.ConstantValue;
 using USFMConverter.Core.Data;
 using USFMConverter.Core.Render;
 using USFMConverter.Core.Util;
 using USFMConverter.UI;
+using USFMToolsSharp.Models.Markers;
 
 namespace USFMConverter.Core
 {
     public class CoreConverter
     {
         public static ICollection<string> supportedExtensions = new List<string> {
-            ".usfm", ".txt", ".sfm"
```

Comment:
Will dropping .txt support affect opening BTT-Writer USFM files?  The chunk files are all .txt.